sdk/storage/Azure.ResourceManager.Storage/api/Azure.ResourceManager.Storage.netstandard2.0.cs

```diff
@@ -757,6 +757,23 @@ public enum AccountStatus
         Available = 0,
         Unavailable = 1,
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct Action : System.IEquatable<Azure.ResourceManager.Storage.Models.Action>
```

Comment:
single word

Reply:
Naming isuses are not covered by this PR. We will take care of this later